docs/concepts/insert.rst

```diff
@@ -21,9 +21,15 @@ using ``delete``. ``Delete`` takes:
 ``from``
   The ``TableSchema`` for the table to delete rows from.
 
+``using``
+  This is a simple ``Query`` that forms the ``USING`` clause of the ``DELETE``
+  statement. This can be used to join against other tables, and the results
+  can be referenced in the ``deleteWhere`` parameters.
```

Comment:
Should we state here that "for simple delete queries, you can set `using = return ()`?